### ISSUE TITLE
need additional #symfony irc channel ops

### ISSUE THREAD

**Issue by jrobeson** [symfony/symfony#14125]
I'm not actually sure where to post this, since the dev list is gone. But i'd like to get additional channel ops for #symfony, #symfony-off, and #symfony-dev.

I'd like to nominate these people:
@WouterJ @Iltar and myself

**Comment by javiereguiluz** [comment]
I occasionally visit the Symfony IRC channels and I always find the three of you helping other people. Therefore, I'm :+1: for this proposal.

By the way, does it make sense to maintain `#symfony-off` and `#symfony-dev` channels?

**Comment by jrobeson** [comment]
@javiereguiluz : #symfony-off is pretty active now. We are really trying to keep off topic and personal discussions in there and not in the main channel

**Comment by jrobeson** [comment]
oh as far as #symfony-dev goes, I'd prefer if it was set to invite only except on hackdays.

**Comment by iltar** [comment]
@jrobeson Thanks for the nomination, I'd be happy to help out as I'm usually online most of the week (work) and sometimes even from home.

@javiereguiluz 99% of the time we redirect people from `#symfony-dev` to `#symfony`.

**Comment by cordoval** [comment]
:+1: for the nomination though i like the freedom irc has to ask any question and bring free speech in discussion as well :older_man:

**Comment by javiereguiluz** [comment]
@jrobeson after reading @iltar comments, what do you think about:

  * Removing `#symfony-dev` and keeping the discussions about Symfony development in the only official site to do that: this GitHub repository.
  * Whenever there is a hackday, the `#symfony` channel will be flooded by Symfony development topics instead of Symfony newcomers questions .... but that isn't necessarily a problem. Newcomers will see how active Symfony development is and some of them may want to join or learn more about this. In addition, hackdays are very rare these days, so we won't disturb `#symfony` channel much.

**Comment by fabpot** [comment]
:+1: ... but I have no idea who can do that.

**Comment by jrobeson** [comment]
@javiereguiluz : I think ti's easy enough to just set the channel to invite only and clear it on hackdays. I'd definitely prefer to keep hackdays separate, but change the topic in #symfony to mention that there is a hackday ongoing.

**Comment by jrobeson** [comment]
here's the current channel ownerships:

#symfony: fabpot, pookey
#symfony-dev: jonwage, Seldaek, lsmith
#symfony-off: pookey, skoop, hal, FuzzyOne

**Comment by jrobeson** [comment]
I'll ask pookey to help us out with #symfony and #symfony-off  and either @lsmith77 or @Seldaek with #symfony-dev.

**Comment by javiereguiluz** [comment]
@jrobeson please reconsider your decission: no one in the Symfony core group monitors the `#symfony-dev` channel. Any discussion about Symfony development done outside this GitHub repository is lost.

And it's confusing to have a `-dev` channel: is that for "Symfony developers" or for "Symfony development"?

**Comment by jrobeson** [comment]
I've been meaning on asking about this for over a year now, but we finally had an instance in which we wanted to kick someone for flooding the channle, but couldn't do it!

**Comment by jrobeson** [comment]
@javiereguiluz : how about we reserve #symfony-hackday channel then? and just leave #symfony-dev invite only to stop people from joining.

**Comment by javiereguiluz** [comment]
I'm a follower of the *"less, but better"* philosophy, so I'd prefer to not create separate channels. However, I'll support any proposal to remove the `#symfony-dev` channel :)

**Comment by jrobeson** [comment]
@javiereguiluz : i'm going ot have to stand firm on using a different channel than #symfony for hack days.

**Comment by iltar** [comment]
@jrobeson what would be the downside of using `#symfony` during the hackdays?

**Comment by jrobeson** [comment]
confusion. Because some people are trying to get help with their actual symfony problem, while other people are trying to work together to fix tickets.

**Comment by jrobeson** [comment]
We already have enough trouble with a situation like this:

<Person1> Can you help me with X.
--- Person2 Joins ---
<Person2> Can you help me with Y.
<Helper> You should do Z.
<Person2> were you telling me to do Z?

**Comment by maschmann** [comment]
Some more flooding :-)
Using a separate channel for hackdays seems a good idea. Does not matter if the channels is empty inbeetween hackdays  - as long as there's a bot or nick parked to prevent overtakes.
Generally a bot in #symfony and/or #symfony-off could solve the ops-problem by a access list and the kick options. Nice side-effect might be to be able to post the Symfony blog posts and community news as they arrive.

**Comment by cordoval** [comment]
if i don't recall wrongly @weaverryan was a separate channel proponent. I agree with him and @jrobeson. What I think I disagree is in terms of freedom, we should let trolling on irc be handled by irc means which is ignoring or inactivity or just time. It has always been the case most of the times but i could be wrong. I am a big proponent of freedom so :-1: for invite onlys. :+1: for separate channel which i think -docs already does for hacking days.

@javiereguiluz is right in which i think github more and more is displacing irc for development and discussion.

**Comment by jrobeson** [comment]
@cordoval : the invite only is just a way to stop people from joining.. the channel will actually be dead, and nobody will be in there.

Also in regards to trolling: Ignoring the trolls is not an effective technique. See the recent #gamergate terribleness

**Comment by iltar** [comment]
What I'd like to prevent mainly, would be spam such as the following:

![IRC Spam](http://i.imgur.com/HRkOW9p.png)

**Comment by cordoval** [comment]
oh wow, ok then :+1:

**Comment by jrobeson** [comment]
what @iltar posted is finally why filed this ticket :)

**Comment by maschmann** [comment]
Otherwise it's always an option to /ignore <people> if they annoy ;-)

**Comment by xabbuh** [comment]
:+1: for giving op permissions to #symfony and #symfony-off, everytime I join the channel, at least one of them is there and they really go a great job

Not sure how to handle #symfony-dev though. Actually, I don't think we really need to do anything about it as there's not much traffic and most people understand to ask support questions in #symfony instead.

**Comment by lsmith77** [comment]
fine for me .. I think I have admin rights on #symfony-dev

**Comment by jrobeson** [comment]
@xabbuh : it seems that at least a few times a week, there's somebody asking questions in #symfony-dev, so Iltar or I have to redirect them to #symfony.

I realized that freenode's ChanServ has a channel redirect feature. That's probably what we should use.

Does everybody agree with redirecting #symfony-dev to #symfony ?

**Comment by jrobeson** [comment]
I think we got a little off topic here? should i file a new issue for that?

**Comment by jrobeson** [comment]
Here are some channel migration instructions i found: https://wiki.ubuntu.com/IRC/MovingChannels

**Comment by WouterJ** [comment]
* I'm fine with the proposed OPs. It's just so that the channel can be cleaned up when needed and with jrobeson, Iltar and me, I think we've 99% of the time covered with at least one OP.
* I'm a big supporter of a different channel for different stuff (that's also why we created #symfony-docs for the sprint day). As shown by jrobeson, multiple questions at the same time are already hard to follow, the amount of messages during a hackday will simply flood the channel, leaving no room for any questions.
* #symfony-dev is a place where devs can find eachother. Maybe not Symfony core devs, but more like devs in the Symfony environment. It's a great place for bundle authors to communicate with eachother. I know of some devs who are in this channel because of this reason and are not in #symfony because of the high activity there.

  However, I also see the people asking in the wrong channel. We should try to remove references to `#symfony-dev` and make the messages more clear (there must be some IRC setting that sents you a private message onces you join a channel iirc)

**Comment by acrobat** [comment]
:+1:  for the new ops, @iltar, @jrobeson and @WouterJ are most of the time online and helped me multiple times!

**Comment by jrobeson** [comment]
I conisder this particular issue closed. I'll opening my other concerns/questions as separate issues.
